### PR TITLE
Add --save-xlsx option

### PR DESCRIPTION
**Pull request by csh519**
Add save xlsx option to CredSweeper.
You can run save xlsx option by adding `--save-xlsx` on command line.
- Added `--save-xlsx` option
- Fix wrong import ordering
- Fix wrong variable using

Below packages are newly included.
- pandas
- openpyxl

@babenek Thanks to solve the pytest mock issue 👍

### CODE REVIEW

**Review by babenek — APPROVED** [review]
Looks good. XLSX report is created.

**Review by mykolaf — APPROVED** [review]
:+1:

File: credsweeper/__main__.py [hunk 1]
```diff
@@ -106,6 +106,12 @@ def get_arguments() -> Namespace:
                         const="output.json",
                         dest="json_filename",
                         metavar="PATH")
+    parser.add_argument("--save-xlsx",
```
Comment (kmnls) [inline credsweeper/__main__.py:109]: In the report generation code either json or xml is generated.
In the argument parsing there is no info about the mutual exclusivity of these options, isn't it?

Comment (csh519) [reply credsweeper/__main__.py:109]: I fixed it by 7b231b5.
Please check.

**Review by kmnls — APPROVED** [review]

**Comment by csh519** [comment]
> Probably, there should be an error message if both are specified, shouldn't be?

@kmnls 
I tested with below command, and it worked fine :)

``` bash
~$ python -m credsweeper --path ~/test/ --save-json json.json --save-xlsx xlsx.xlsx
```

**Pull request merged**